modules/odc-creating-vms.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="odc-creating-vms_{context}"]
+= Using the Developer perspective to create a virtual machine
+
+You can create a virtual machine by using the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* A cluster administrator has installed the {VirtProductName} Operator in your cluster.
+* You have logged in to the {product-title} web console and are in the *Developer* perspective.
+
+.Procedure
+
+. In the *+Add* view, click on the *Virtual Machines* tile to see the *Create new VirtualMachine* page.
+
+. Select a virtual machine type from the *Template catalog*.
+
+. Click *Quick create VirtualMachine* to quick create a virtual machine, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.
++
+[NOTE]
+====
+For Microsoft Windows virtual machines, only the *Customize VirtualMachine* option is available.
+====
+
+. If you choose to customize your virtual machine, the *Customize Template parameters* page allows you to modify *Storage* and *Optional Parameters* settings. When you have finished customizing your virtual machine settings, click *Next* to see the *Review and create VirtualMachine* page. Review your settings, then click *Create VirtualMachine*.
```

Comment:
```suggestion
. If you choose to customize your virtual machine, you can modify *Storage* and *Optional Parameters* settings on the *Customize Template parameters* page. When you have finished customizing your virtual machine settings, click *Next* to see the *Review and create VirtualMachine* page. Review your settings, then click *Create VirtualMachine*.
```
- Avoid saying that inanimate things "allow" or "enable" humans to do tasks (anthropomorphism).